README.md

```diff
@@ -34,8 +34,8 @@ Installs the libraries and the necessary include files.
 The include files are placed in a brotli subdir under the install prefix so
 include them like
 
-	#include <brotli/decode.h>
-	#include <brotli/encode.h>
+	#include <brotli/include/brotli/decode.h>
+	#include <brotli/include/brotli/encode.h>
```

Comment:
After `make install`, the include files are not in that path. They're installed in $prefix/include/brotli/